android/app/build.gradle

```diff
@@ -135,12 +136,39 @@ android {
             }
         }
     }
+    dataBinding {
+        enabled true
+    }
 }
 
+project.ext.vectoricons = [
+    iconFontNames: [ 'MaterialIcons.ttf' ] // Name of the font files you want to copy
```

Comment:
The comment is superfluous, the name of the property is descriptive enough.

Reply:
done!